### ISSUE TITLE
Somehow allow multiple "after" examples

### ISSUE THREAD

**Issue by isomorpheme** [haskell/error-messages#73]
See e.g. #65 where there's two possible solutions, so as a workaround I made `before/Main1.hs` and `before/Main2.hs` which are identical, but `after/Main1.hs` and `after/Main2.hs` which are different.

**Comment by PPKFS** [comment]
How about having both examples in the same `after`?

something like

```
aFixedOneWay = ...

aFixedTheOther = ...
```

**Comment by isomorpheme** [comment]
Yeah that's another workaround I considered - the issue there would be if we eventually want to have before/after diffs, this wouldn't work as well.

**Comment by david-christiansen** [comment]
What if we tweaked the generator to treat any directory that matches the glob `after*` as a potential "after", and then multiplexed them appropriately?
